roxie/ccd/ccdcontext.cpp

```diff
@@ -770,7 +770,7 @@ class CRoxieWorkflowMachine : public WorkflowMachine
                     MilliSleep(PERSIST_LOCK_SLEEP + (getRandom()%PERSIST_LOCK_SLEEP));
                     persistLock.setown(getPersistReadLock(goer));
                 }
-                Owned<IDistributedFile> f = wsdfs::lookup(goer, queryUserDescriptor(), AccessMode::tbdWrite, false, false, nullptr, defaultPrivilegedUser, INFINITE);
+                Owned<IDistributedFile> f = wsdfs::lookup(goer, queryUserDescriptor(), AccessMode::writeSequential, false, false, nullptr, defaultPrivilegedUser, INFINITE);
```

Comment:
Do we ever write other than sequentially?

Reply:
we don't currently (now that index writes have been changed)